src/app/components/AppHeader/AppHeader.module.css

```
@@ -15,6 +15,12 @@
   margin: 0;
   padding: 0.2em 0.5em;
   font-size: 0.8em;
+  color: var(--color-text-secondary);
+  font-family: var(--font-family-primary);
+}
+
+.ticker {
+  color: #f09f12;
```

mabye using this colour, its close and already existing: 
```suggestion
  color: var(--gradient-primary-from);
```

---

It's close, but different. 
If we had a different theme, would we change the logo too?
Maybe :-).
Still, I think it's a good idea -> replaced it.